docs/modules/install/pages/index.adoc

```diff
@@ -96,8 +96,8 @@ You can configure it with `crontab -e`, for instance if you've created your Deci
 
 [source,console]
 ----
-# Remove expired data portability files
-0 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:delete_data_portability_files
+# Remove expired download your data files
+0 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:delete_download_your_data_files
```

Comment:
I've just realized that we'll need to add this change in the crontab on the CHANGELOG for already existing apps. Can you add it please? 

(I didn't catch it when doing the Acceptance Criteria, sorry!)